spec/rspec/scenarios_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+require 'capybara/rspec'
+
+RSpec.configuration.before(:each, { file_path: "./spec/rspec/features_spec.rb" } ) do
+  @in_filtered_hook = true
+end
+
+if RSpec::Core::Version::STRING.to_f >= 3.0 || RUBY_ENGINE == 'rbx'
```

Comment:
I have no idea why 'rbx' would cause a failure here, and really don't want to skip testing on it without understanding exactly why it's failing